src/services/github.ts

```diff
@@ -137,6 +140,32 @@ export default class GithubService {
     return this.octokit.repos.getBranch(criteria);
   }
 
+  public createIssue(
+    criteria: RestEndpointMethodTypes['issues']['create']['parameters']
+  ): Promise<RestEndpointMethodTypes['issues']['create']['response']> {
+    return this.octokit.issues.create(criteria);
+  }
+
+  public updateIssue(criteria: {
+    owner: any;
```

Comment:
I think we have a type we can type check criteria against.  If not can we convert this into a type and use it here?  Most of these are strings.  Can we be more restrictive on the type (e.g., owner is of type string instead of any)?